website/static/main.css

```diff
@@ -72,6 +72,39 @@ pre {
 .header-anchor { font-size: 100%; visibility: hidden;}
 h1:hover a, h2:hover a, h3:hover a, h4:hover a { visibility: visible}
 
+/* seaarchBox */
+.searchBox {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+input[type="search"]{
+    height: 30px;
+  line-height: 30px;
```

Comment:
tiny nit: let's indent all of the code equally

Reply:
Alright